google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/util/AppEngineUtil.java

```diff
@@ -16,34 +16,64 @@
 
 package com.google.cloud.tools.intellij.appengine.util;
 
-import org.joda.time.DateTime;
-import org.joda.time.format.DateTimeFormat;
-import org.joda.time.format.DateTimeFormatter;
+import com.google.gson.Gson;
+import com.google.gson.JsonParseException;
+import com.google.gson.reflect.TypeToken;
+
+import java.lang.reflect.Type;
+import java.util.List;
 
 /**
  * App Engine utility methods
  */
 public class AppEngineUtil {
 
+  // Just to return two strings.
+  public static class VersionService {
```

Comment:
I don't think you need this. If you like, you could make a helper getters in your DeploymentOutput class to give you these values.